docs/blender_docs/scene_gltf2.rst

```diff
@@ -26,11 +26,13 @@ This importer/exporter supports the following glTF 2.0 features:
 - Textures
 - Cameras
 - Punctual lights (point, spot, and directional)
+- Extensions (listed below)
+- Extras (custom properties)
```

Comment:
nit: consider using the Blender terminology first, "Custom Properties (extras)"

Reply:
I'm not sure about this one, because this list is introduced as a list of "supported glTF 2.0 features," not Blender features.